packages/core/src/codewhisperer/models/model.ts

```diff
@@ -88,7 +88,7 @@ export class CodeSuggestionsState {
         return (this.#instance ??= new this())
     }
 
-    protected constructor(context: vscode.Memento = globals.context.globalState, fallback: boolean = false) {
+    protected constructor(context: vscode.Memento = globals.context.globalState, fallback: boolean = true) {
```

Comment:
ci failure might be related to this change. tests may need to be updated

Reply:
tests fixed.